dask/array/creation.py

```diff
@@ -201,7 +201,7 @@ def linspace(start, stop, num=50, endpoint=True, retstep=False, chunks='auto',
         The number of samples on each block. Note that the last block will have
         fewer samples if `num % blocksize != 0`
     dtype : dtype, optional
-        The type of the output array. Default is given by ``numpy.dtype(float)``.
```

Comment:
Are there cases where `numpy.dtype(float)` is not the default?

Reply:
I don't think so.  I removed this at first because I was going to just define `dtype=float` in the signature.  I then saw the implementation and noticed that it was calling `np.linspace`.  I decided just to stick with that in case there was some 32bit issue (which we don't currently test).  I didn't see a strong reason to put this back in afterwards.  Happy to do so though if desired.

Reply:
Okay, I was just curious for my own knowledge moving forward. Happy with this as is